apps/academy/src/pages/tracks/nft-solidity/5.mdx

```diff
@@ -320,62 +339,360 @@ Now that we have our application setup, we can edit the Index Page.
 ### The Index Page
 
```

Comment:
You're right, "We'll only make one change to this file" is not accurate.

I updated the explanation and clarified what the resulting file should look like. Commit: https://github.com/Developer-DAO/academy-turbo/pull/131/commits/7cc4f2fb52e9c67216f2830500c600064fb8c243

I didn't include the initial version you pasted in above. Do you think we should add this?

(The main reason I left out was to avoid having to keep that up-to-date since that file changes a lot on each version of Next.js):

Reply:
> I didn't include the initial version you pasted in above. Do you think we should add this?
No, don't add it. I just gave it for reference 😁 

Can you check my comments in your commit: https://github.com/Developer-DAO/academy-turbo/commit/7cc4f2fb52e9c67216f2830500c600064fb8c243 - I hope my work flow is okay here and not bouncing you around too much. Lmk if I can improve the workflow.

Reply:
Thanks for your feedback on my commit, I made the change.